tests/test_streamflow.py

```diff
@@ -0,0 +1,27 @@
+from xclim import streamflow
+import numpy as np
+
+
+def test_base_flow_index(ndq_series):
```

Comment:
Tests seem somewhat sparse here testing only for metadata additions and not output values

Reply:
There is an output test in test_indices.